app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    masks = 0
+    for visitor in friends:
+        try:
+            cafe.visit_cafe(visitor)
+        except VaccineError:
+            return "All friends should be vaccinated"
+        except NotWearingMaskError:
+            masks += 1
+    if masks:
+        return f"Friends should buy {masks} masks"
```

Comment:
Here you are comparing `masks` to `0` which is not necessary. You can simplify this by just checking `if masks:`. This is more pythonic and efficient.